### ISSUE TITLE
Add params object[] overload to SetArgDisplayNames

### ISSUE THREAD

**Issue by jnm2** [nunit/nunit#3360]
My recommendation in [this comment](https://github.com/nunit/nunit3-vs-adapter/issues/650#issuecomment-525906101) feels a little bulky:

```cs
tcd.SetArgDisplayNames(testData.Name, testData.Gender.ToString(), testData.Age.ToString());
```

I'd like to add an overload that allows passing any objects:

```cs
tcd.SetArgDisplayNames(testData.Name, testData.Gender, testData.Age);
```

It could simply .ToString() all the arguments, but it would be even better IMO to format the objects just as in the default test name. E.g. if an object is a `byte` and has the value 255, we show `byte.MaxValue` instead of showing .ToString().

String.Join has overloads for either object or string, and I added an analogous overload to Humanizer a while back which I've liked.

**Comment by stevenaw** [comment]
I've labeled this as a good first issue

**Comment by OsirisTerje** [comment]
Can be a good test to see if CoPilot can fix it.....

**Comment by stevenaw** [comment]
True, perhaps copilot could too.

Though we don't have too many issues in the repo for people to pick up to contribute and get involved with the project. I had thought it could be good to set some aside for interested individuals. It's how I first became involved 😀

**Comment by OsirisTerje** [comment]
Yeah, agree, I just was so curious about the agent being used for coding this way.   But I promise - I will not do it again on "Good first issues " ;-)
